libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -896,11 +896,13 @@ TEST_CASE("ring_buffer_async_query", "[execution_context]")
     struct _completion
     {
         uint8_t* buffer = nullptr;
+        size_t consumer_offset;
```

Comment:
does this need to be initialized to 0?

Reply:
Also, do we want to add any sort of validation check in the below (i.e is it supposed to always be > 0)?

Reply:
No, it's an output variable, and then used when calling the actual existing tests with the returned current value from ebpfcore.